### PR TITLE
Clarify SBD STONITH resource step

### PR DESCRIPTION
**Pull request by tahliar**
### PR creator: Description

Per DOCTEAM-834, step 4 was a little confusing as there's a lot of info there and it's not immediately obvious that this is the step where you configure the resource (at least until you scroll past and see that there's nothing else after it). 

I thought the fix would be more complex, but actually all I ended up doing was moving the context from the start of the procedure to the appropriate steps. It's still not an ideal structure for a step in a procedure, but for the purposes of this issue (making clear what the step actually is), this fix does the trick. 

### PR creator: Are there any relevant issues/feature requests?

* jsc#DOCTEAM-834

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE-HA 15
  - [x] 15 next *(current `main`, no backport necessary)*
  - [x] 15 SP4
  - [x] 15 SP3
  - [x] 15 SP2
  - [x] 15 SP1
- SLE-HA 12
  - [x] 12 SP5
  - [x] 12 SP4

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

**Comment by tahliar** [comment]
Original procedure: https://documentation.suse.com/sle-ha/15-SP4/html/SLE-HA-all/cha-ha-storage-protect.html#pro-ha-storage-protect-fencing

New procedure: 

image

File: xml/ha_storage_protection.xml [hunk 1]
```diff
@@ -804,13 +783,18 @@ SBD_WATCHDOG_TIMEOUT=5</screen>
...
        SBD was set to <literal>30</literal> seconds.</para>
      </callout>
    </calloutlist>
   </step>
   <step xml:id="st-ha-storage-protect-fencing-static-random">
+   <para>
+    Configure the SBD &stonith; resource. You do not need to clone this resource.
+   </para>
    <para>
     For a two-node cluster, in case of split-brain, there will be fencing issued from
```
Comment (dariavladykina) [inline xml/ha_storage_protection.xml:799]:
```suggestion
    For a two-node cluster, in case of split brain, there will be fencing issued from
```

**Review by dariavladykina — COMMENTED** [review]
Hi Tahlia, LGTM, but I couldn't help updating the 'split brain' as it is a noun here :-)

File: xml/ha_storage_protection.xml (hunk 1 above)
Comment (tahliar) [reply xml/ha_storage_protection.xml:799]: Thanks!

**Pull request merged**